webview-codicons-sample/media/styles.css

```diff
@@ -56,5 +56,44 @@ h1 {
 
 #icons .icon .codicon {
 	font-size: 32px;
-	padding-bottom: 16px;
+}
+
+/* Codicon mouse events demo */
+
+div.iconBar .codicon {
+	font-size: 60px;
+	cursor: pointer;
+}
+
+div.iconBar .disabled {
+	opacity: 0.3;
+	cursor: not-allowed;
+}
+
+/* Codicons in buttons demo */
+
+div.iconButtonBar .iconButton .codicon {
+	font-size: 40px;
+	padding: 3px;
+}
+
+div.styledIcon .codicon {
+	font-size: 50px;
+	color: green;
```

Comment:
this is referencing a color that isn't a color token, something we avoid doing. Something like `var(--vscode-debugIcon-startForeground` would be preffered.

Reply:
That's a really good point. Fixed.